route/pasteBinRoute.js

```diff
@@ -1,11 +1,12 @@
 const express = require('express');
 const app = express();
 const router = express.Router();
-const {createPasteBin , getPasteBin} = require('../controller/pasteBinController')
+const {createPasteBin , getPasteBin} = require('../controller/pasteBinController');
+const {authToken} = require('../middleware/authorization')
 
-router.post('/paste', createPasteBin);
+router.post('/paste', authToken , createPasteBin);
```

Comment:
`isLoggedIn` is a better name than `authToken`

Reply:
Ok sir